APE_Black_Formatting.rst

```diff
@@ -0,0 +1,307 @@
+.. _Black: https://github.com/psf/black
+.. _Django DEP-0008: https://github.com/django/deps/blob/main/final/0008-black.rst
+.. _pandas pre-commit: https://pandas.pydata.org/docs/development/contributing_codebase.html#pre-commit
+.. _coding style: https://docs.astropy.org/en/latest/development/codeguide.html#coding-style-conventions
+
```

Comment:
[PEP 8](https://peps.python.org/pep-0008/) is referred to in the text, there's no reason not to list it here.
```suggestion
.. _PEP8: https://peps.python.org/pep-0008/

```